README.md

```diff
@@ -22,93 +22,97 @@ you can find more examples in the **[examples](/tabled/examples/)** folder.
 
 ## Table of Contents
 
-- [Usage](#usage)
-- [Settings](#settings)
-  - [Style](#style)
-    - [Styles](#styles)
-      - [ascii](#ascii)
-      - [modern](#modern)
-      - [sharp](#sharp)
-      - [rounded](#rounded)
-      - [extended](#extended)
-      - [psql](#psql)
-      - [markdown](#markdown)
-      - [re\_structured\_text](#re_structured_text)
-      - [dots](#dots)
-      - [ascii\_rounded](#ascii_rounded)
-      - [blank](#blank)
-      - [empty](#empty)
-    - [Customization](#style-customization)
-    - [Cell Border](#cell-border)
-    - [Text on borders](#text-on-borders)
-    - [Colorize borders](#colorize-borders)
-    - [Theme](#theme)
-      - [Colorize content](#colorize-content)
-      - [Column names](#colorize-content)
-  - [Alignment](#alignment)
-  - [Format](#format)
-  - [Padding](#padding)
-  - [Margin](#margin)
-    - [Color](#margin-color)
-  - [Shadow](#shadow)
-  - [Width](#width)
-    - [Truncate](#truncate)
-    - [Wrapping](#wrapping)
-    - [Increase width](#increase-width)
-    - [Justify](#justify)
-    - [Priority](#priority)
-    - [Percent](#percent)
-  - [Height](#height)
-    - [Height Increase](#height-increase)
-    - [Height Limit](#height-limit)
-  - [Rotate](#rotate)
-  - [Disable](#disable)
-  - [Extract](#extract)
-  - [Header and Footer and Panel](#header-and-footer-and-panel)
-  - [Merge](#merge)
-  - [Concat](#concat)
-  - [Highlight](#highlight)
-  - [Span](#span)
-    - [Horizontal span](#horizontal-span)
-    - [Vertical span](#vertical-span)
-  - [Split](#split)
-  - [Duplicate](#duplicate)
-- [Derive](#derive)
-  - [Override a column name](#override-a-column-name)
-  - [Hide a column](#hide-a-column)
-  - [Set column order](#set-column-order)
-  - [Format fields](#format-fields)
-  - [Format headers](#format-headers)
-  - [Inline](#inline)
-- [Table types](#table-types)
-  - [`Table`](#table)
-  - [`IterTable`](#itertable)
-  - [`CompactTable`](#compacttable)
-  - [`PoolTable`](#pooltable)
-  - [`ExtendedTable`](#extendedtable)
-- [Tips and Tricks](#tips-and-tricks)
-  - [`std::fmt::*` options](#std::fmt::*-options)
-  - [Tuple combination](#tuple-combination)
-  - [Tuple options](#tuple-options)
-  - [Object](#object)
-  - [Builder](#builder)
-  - [Macros](#macros)
-    - [`col` and `row`](#col-and-row)
-    - [`static_table`](#static_table)
-- [Features](#features)
-- [Formats](#formats)
-  - [`json` format](#json-format)
-  - [`ron` format](#ron-format)
-  - [`csv` format](#csv-format)
-  - [`toml` format](#toml-format)
-  - [`html` format](#html-format)
-- [Notes](#notes)
-  - [Charset](#charset)
-  - [ANSI escape codes](#ansi-escape-codes)
-  - [Emoji](#emoji)
-  - [Semver](#semver)
-  - [MSRV](#msrv)
-  - [Comparison](#comparison)
+- [    tabled ](#----tabled-)
+  - [Table of Contents](#table-of-contents)
+  - [Usage](#usage)
```

Comment:
unsure about this approach

@howjmay, can you elaborate why this is better than the usual

```md
# title

## subtitle
```

Reply:
I could guess that it's auto formatting.
I had got it a few times.

@Alexdelia thanks for the look

Reply:
I could imagine it's an auto formatting.
Thanks for the look @Alexdelia